dbt/adapters/bigquery/gcloud.py

```diff
@@ -1,6 +1,8 @@
-from dbt.events import AdapterLogger
+from typing import Optional, Tuple
+
 import dbt.exceptions
-from dbt.clients.system import run_cmd
+from dbt.clients import system
+from dbt.events import AdapterLogger
```

Comment:
- Are you using anything from `dbt.clients.system` besides `run_cmd`? Or is this a preference for namespacing `system.run_cmd` where it's called below?
- Did you end up needing `AdapterLogger`? If not, could you remove the import?